src/OpenCommandLine/OpenCommandLinePackage.cs

```diff
@@ -102,6 +113,8 @@ private void OpenCustom(object sender, EventArgs e)
             string confPlatform = VsHelpers.GetSolutionConfigurationPlatformName(_dte);
             arguments = arguments.Replace("%platform%", confPlatform);
 
+            arguments = arguments.Replace("%file%", string.Empty);
```

Comment:
Any configure %folder% parameter need to be remove when just opening the console